invite/test/e2e.test.ts

```diff
@@ -63,14 +63,15 @@ describe('end-to-end', () => {
     db.destroy();
   });
 
+  beforeEach(() => nock.cleanAll());
```

Comment:
If the `cleanAll` call lives in the `afterEach` block, there are cases where a test can fail and the error throws itself out of the test before the `cleanAll` call executes, which leaves un-requested nocks in place and fails all tests that follow. Doing it before each test is more reliable in the event of an error within a test.

Reply:
A cleaner solution is to put this in the `afterEach`:
```typescript
// My preferred syntax
try {
  if (!nock.isDone()) {
    throw new Error('Not all nock interceptors were used!');
  }
} finally {
  nock.cleanAll();
}
```

or
```typescript
// Arguably more straightforward, arguably more confusing syntax
const nockWasDone = nock.isDone();
nock.cleanAll();
if (!nockWasDone) {
  throw...
}
```

(here and in the other test file)
This way you don't have to double-clean the nocks :)

Reply:
I tried that and it didn't work. I fiddled with this for a while, Jest seems to be picky with nocks and errors. It's not double-cleaning the nocks, it's just cleaning before each test instead of after, which seems fine to me, so ¯\\_(ツ)_/¯